scripts/merkle/dapiManagementMT.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
+const path = require('path');
+const { signEIP712Message, constructMerkleTree } = require('./merkle-utils');
+const ethers = require('ethers');
+
+// Reading the input data
+const inputData = JSON.parse(fs.readFileSync(path.join(__dirname, 'treeValues', 'dapiManagementMT.json'), 'utf8'));
+const values = inputData.values;
+
+const tree = constructMerkleTree(values, ['string', 'bytes32', 'address']);
```

Comment:
While working on `DapiDataRegistry` I started thinking that is nice to have `dapiName: 'API3/USD'` string in the dapiManagementMT.json metadata file but shouldn't we construct the merkle tree with the `bytes32` string instead since even the `DapiServer.sol` is expecting to received the formatted `bytes32` string when calling `SetDapiName()`?
```suggestion
const tree = constructMerkleTree(values, ['bytes32', 'bytes32', 'address']);
```

Reply:
Yes, the contract wouldn't be able to convert `string` to `bytes32` aynway

Reply:
I guess it is now possible to do something like the following to convert it but it would consume more gas as well:
`api3ServerV1.setDapiName(bytes32(abi.encodePacked(dapiName)), dataFeedId);`

Reply:
`bytes32(abi.encodePacked(stringVariable))` truncates the string down to 32 characters so that's not really an option imo